internal/cmd/shim.go

```diff
@@ -58,6 +58,7 @@ func pluginSettings(r *compiler.Result, cs config.CombinedSettings) *plugin.Sett
 		Overrides: over,
 		Rename:    cs.Rename,
 		Codegen:   pluginCodegen(cs.Codegen),
+		Plugin:    pluginPlugin(cs),
```

Comment:
As discussed offline, probably better to move the elements of this into the existing `Codegen` field.